src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const thead = document.querySelectorAll('thead tr');
+const tbody = document.querySelectorAll('tbody tr');
+const tfoot = document.querySelectorAll('tfoot tr');
+
+const columnIndex = 1;
+
+thead.forEach((tr) => {
+  const th = tr.children[columnIndex];
+  const clonedTh = th.cloneNode(true);
+
+  tr.insertBefore(clonedTh, tr.children[columnIndex + 3]);
```

Comment:
The insertion logic assumes that there are at least four columns in each row (index `columnIndex + 3`). If a row has fewer columns, this will result in an error. Consider checking the length of `tr.children` before attempting to insert the cloned node to ensure the target index exists.